<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #6a11cb, stop:1 #2575fc);
</string>
  </property>
  <widget class="QFrame" name="box_login">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>881</width>
     <height>711</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 180); /* Transparansi */
border-radius: 15px; /* Membuat sudut melengkung */
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTextBrowser" name="selamat_datang">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>851</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:7.8pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:28pt;&quot;&gt;Selamat Datang Di Aplikasi SAKAPEUNG&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:14pt;&quot;&gt;Sistem Aplikasi Kereta Api Pemesanan Endah Unggul&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="login" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>801</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : white;</string>
    </property>
    <widget class="QTextBrowser" name="tulisan_login">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell Extra Bold</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Rockwell Extra Bold'; font-size:7.8pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:28pt;&quot;&gt;LOGIN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="icon_username">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>icons/username.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="icon_password">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>240</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>icons/pw.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="email_input">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>491</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">    border: none;
    border-bottom: 2px solid black;
	border-radius: 0px;
    color: black;
    font-size: 16px;
    padding: 5px;
    background: transparent;</string>
     </property>
     <property name="placeholderText">
      <string>masukkan email anda.....</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_input">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>230</y>
       <width>491</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">    border: none;
    border-bottom: 2px solid black;
	border-radius: 0px;
    color: black;
    font-size: 16px;
    padding: 5px;
    background: transparent;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>masukkan password anda.....</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="nanya">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>420</y>
       <width>256</width>
       <height>31</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Belum Punya Akun?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="masuk_ke_aplikasi">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>340</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #007BFF;
    color: white;
    border-radius: 10px;
    padding: 8px;
}
QPushButton:hover {
    background-color: #0056b3;
}
</string>
     </property>
     <property name="text">
      <string>Masuk Ke Aplikasi</string>
     </property>
    </widget>
    <widget class="QPushButton" name="registrasi">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>420</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #007BFF;
    color: white;
    border-radius: 10px;
    padding: 8px;
}
QPushButton:hover {
    background-color: #0056b3;
}
</string>
     </property>
     <property name="text">
      <string>Registrasi</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
